tests/e2e/adu/device/e2e_device_commands.c

```diff
@@ -85,9 +85,8 @@ static uint16_t usReceivedTelemetryPubackID;
 #define e2eADU_UPDATE_PROVIDER        "ADU-E2E-Tests"
 #define e2eADU_UPDATE_NAME            "Linux-E2E-Update"
 #define e2eADU_UPDATE_VERSION         "1.0"
-#define e2eADU_UPDATE_VERSION_NEW     "1.1"
+#define e2eADU_UPDATE_VERSION_NEW     "1.2"

```

Comment:
Are we going to leave this at 1.2?

Reply:
Interesting question - I suppose with the unique update name, there's no need to have 1.2 for these other than to denote for us that it's a v5 manifest. I'll revert them!

Reply:
Done!